sim/src/main/cc/midasexamples/PlusargsModule.h

```diff
@@ -0,0 +1,135 @@
+// See LICENSE for license details.
+
+#include "simif_peek_poke.h"
+
+#include "bridges/plusargs.h"
+
+#include <iomanip>
+#include <iostream>
+
+/**
+ * @brief Plusargs Bridge Driver Test
+ *
+ * This test parses `+plusargs_test_key` (given by TutorialSuite.scala)
+ * and asserts for the correct values
+ */
+class PlusargsModule_t : public simif_peek_poke_t {
+private:
+  /**
+   * Find the `+plusargs_test_key` and set the appropriate member variables
+   * All macros should be passed to this constructor.
+   *
+   * The value is checked to reject any runtime values that are too big for the
+   * @param [in] args The argv as a vector
+   */
+  void parse_key(const std::vector<std::string> &args) {
+    const std::string find_key = "+plusargs_test_key=";
```

Comment:
This could be `const static std::string_view`.

Reply:
Would you prefer `constexpr` over `const static` ?